module-4/backend/week_three.md

```diff
@@ -5,15 +5,32 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+Node is a JavaScript runtime environment that can be used server or client side. 
 2. What is Express? Why is Express similar to in the Ruby world?
+Express is a Node.js framework that provides features for web applications. There are many frameworks 'build on top' of express. Rails is similar to express in the ruby world
```

Comment:
I would say Express is closer to Sinatra than Rails because it is lightweight and un-opinionated.